libraries/Microsoft.Bot.Connector/Authentication/SkillValidation.cs

```diff
@@ -86,6 +86,11 @@ public static bool IsSkillClaim(IEnumerable<Claim> claims)
         {
             var claimsList = claims.ToList();
 
+            if (claimsList.Any(c => c.Value == AuthenticationConstants.AnonymousSkillAppId && c.Type == AuthenticationConstants.AppIdClaim))
```

Comment:
consider: would it be valuable to have this validation built in in a single place? SkillValidation.IsAnonymous(claims)?

Reply:
Maybe in a future rev? I am trying to reduce the new surface area, we can add this helper once we prove this solution works.